template/src/ontology/Makefile.jinja2

```diff
@@ -634,7 +634,7 @@ pattern_schema_checks dosdp_validation: $(TMPDIR)/pattern_schema_checks
 
 .PHONY: update_patterns
 update_patterns: download_patterns
-	cp -r $(TMPDIR)/dosdp/*.yaml $(PATTERNDIR)/dosdp-patterns
+	if [ -d $(TMPDIR)/dosdp ]; then if [ "$(ls $(TMPDIR)/dosdp)" ]; then cp -r $(TMPDIR)/dosdp/*.yaml $(PATTERNDIR)/dosdp-patterns; fi; fi
```

Comment:
This works on the terminal but not calling `make update_patterns`. I don't know why.

Reply:
Because the `$(ls ...)` is interpreted by Make – the shell never sees it. You need to escape the `$` character so that it is passed to the shell: `$$(ls $(TMPDIR)/dosdp)`.

Reply:
Ah, thanks, Damien!